### DISCUSSION TITLE
Error when installing TDB_full_Hotfixes in HeidiSQL

### DISCUSSION THREAD

**Discussion by Dreyzx** [TrinityCore/TrinityCore#28270] [General]
Hi when i install TDB_full_Hotfixes in HeidiSQL i have an error near the end say "No mapping for the unicode character exists in the target multi-byte code page".
I can't install Worldserver because of that i don't know what to do thank you for help !
![Capture](https://user-images.githubusercontent.com/111538415/191615087-feb4a5c4-8193-4fc6-a80a-33a64c3e38eb.PNG)

**Comment by Aokromes** [comment]
don't use heidy, leave core the job of import databases as stated on wiki.

**Comment by shamage** [comment]
You have 2 options that currently work without problems:

1.) DB implementation via MySQL - command client
2.) you let it be automatically implemented as described in the TC Wiki guide
